app/models/api/area.rb

```diff
@@ -43,6 +43,10 @@ def self.all_by_area_code
     end
   end
 
+  def self.derived_datasets
+    all_by_area_code.sort_by { |area_code, _| area_code }.map { |_, area| area }.find_all { |area| area.is_derived? }
```

Comment:
This line is a little to long.

You can add newlines:

````ruby
all_by_area_code
  .sort_by { |area_code, _| area_code }
  .map { |_, area| area }
  .find_all { |area| area.is_derived? }
````

.. but that's just partially solving it. I think this can be rewritten to:

````ruby
all_by_area_code.sort.map(&:last).find_all(&:is_derived?) # I think instead of `find_all` you can also use `select` but I think that they work similar on arrays?
````